### ISSUE TITLE
Cannot find `git` executable on on `windows-2022` (GitHub-hosted runner)

### ISSUE THREAD

**Issue by ochococo** [webfactory/ssh-agent#136]
Observed error on `windows-2022` ([GitHub-hosted runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources)) that `git` command cannot be found:

**Config:**

```yaml
- uses: actions/checkout@v2
- uses: webfactory/ssh-agent@v0.5.4
    with:
    ssh-private-key: |
        ${{ secrets.[redacted] }}
        ${{ secrets.[redacted] }}
```

**Result:**

```bash
'git' is not recognized as an internal or external command,
[198](https://github.com/[redacted]#step:4:199)
operable program or batch file.
[199](https://github.com/[redacted]#step:4:200)
Error: Command failed: git config --global --replace-all url."git@key-[redacted].[redacted]".insteadOf "https://github.com/[redacted]"
[200](https://github.com/[redacted]#step:4:201)
'git' is not recognized as an internal or external command,
[201](https://github.com/[redacted]#step:4:202)
operable program or batch file.
```

**Comment by ochococo** [comment]
@mpdude We could consider using something like that:

```
const io = require('@actions/io');
gitCmd = await io.which('git.exe')
```

My js skills are too rusty to attempt it 🛑

**Comment by ochococo** [comment]
Tested `webfactory/ssh-agent@8a9e20a5860cb7f6b3a1d92a74799581e054d850` and can confirm it works:

<img width="200" alt="image" src="https://user-images.githubusercontent.com/3382607/196698683-3e8ae7d8-8872-41b8-aabe-862ef2cc9248.png">

**Comment by mpdude** [comment]
Released v0.7.0.
